fab/services/templates/celery_main_bash.sh

```diff
@@ -0,0 +1,21 @@
+# Catch any TERM or INT signals and propagate
```

Comment:
i kind of liked it better when it was just one file to add every time we add a new worker instead of two. could we make a generic shell script that gets invoked from supervisor and passes all of the args it receives to ./manage.py celery worker?